DevIL/src-IL/include/il_endian.h

```diff
@@ -39,8 +39,10 @@
 	#define BigDouble(d)  
 #else
 	#undef __BIG_ENDIAN__
+#if !defined(__LITTLE_ENDIAN__) 
 	#undef __LITTLE_ENDIAN__  // Not sure if it's defined by any compiler...
```

Comment:
Yes, anything else? once approved I'll run it on a few other llvm machines to make sure it builds as expected.